### ISSUE TITLE
When i change a character by one i make it crash

### ISSUE THREAD

**Issue by SaulexAldox** [ShadowMario/FNF-PsychEngine#8460]
### Describe your bug here.

ex: i choose fresh, then press 7, then change the character to the one i maked, and then i press enter, and the engine crashes

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

maked a character in mods and add images

**Comment by SaulexAldox** [comment]
![characters 03_05_2022 11_44_14 a  m  (2)](https://user-images.githubusercontent.com/103857168/166500677-3e463be9-c2b8-439e-8d8b-a3f91966f558.png)
![characters 03_05_2022 11_44_02 a  m  (2)](https://user-images.githubusercontent.com/103857168/166500681-c3e09c40-70aa-4ff2-a7e6-8dd8297f1b89.png)
![characters 03_05_2022 11_44_08 a  m](https://user-images.githubusercontent.com/103857168/166500683-5285feb9-03c7-4a96-aa23-7a9a461f080d.png)

**Comment by frantastic24** [comment]
Not an image of the files, the actual files

**Comment by SaulexAldox** [comment]
![Fotos 03_05_2022 12_12_29 p  m  (2)](https://user-images.githubusercontent.com/103857168/166504578-04c4b4d8-4535-4df7-a8a4-98d2d9f87b02.png)

**Comment by frantastic24** [comment]
Put them on a zip

**Comment by SaulexAldox** [comment]
https://www.mediafire.com/file/nrgg1mqyn7prcoy/soneec.zip/file

**Comment by frantastic24** [comment]
it works fine and everything seems correct other than the massive file size, are you sure everything is in the right folder
![image](https://user-images.githubusercontent.com/95725102/166520242-5eb49c99-0155-4477-acbc-15c9f52e77e6.png)

**Comment by SaulexAldox** [comment]
yeah, json on character folder (mods) images/character where the images belong and images/icons where is the icon, maybe is the psych engine version? idk

**Comment by frantastic24** [comment]
Not sure how important engine version is but can't hurt to try a newer one, does this happen in any song? Does it also happen in pixel stages (memory issue maybe)? I'm assuming it works fine in character editor

**Comment by SaulexAldox** [comment]
yeah its work fine

**Comment by SaulexAldox** [comment]
yeah its a version issue, thanks!
